src/client/components/GetMeal.js

```diff
@@ -0,0 +1,129 @@
+import AddReservation from "./AddReservation";
+import React, { useState, useEffect } from "react";
+import { useParams} from 'react-router-dom';
+import MealsImages from "./MealsImages";
+
+const GetMeal = ()=>{
+    const [meal, setMeal] = useState({});
+    const [reservation, setReservation] = useState(false);
+    const [availableReservations, setAvailableReservations] = useState(null)
+    const [totalGuests, setTotalGuests] = useState(null)
+    const [message, setMessage] = useState("");
+    const [error, setError] = useState(null);
+    const [loading, setLoading] = useState(true);
+    
+    const params = useParams();
+    useEffect(() =>{
+        const fetchingMealsApi = async() => {
```

Comment:
Here you have switched to using async. Then later you go back to using the .then method.